dashboard/app/controllers/survey_results_controller.rb

```diff
@@ -14,9 +14,18 @@ def create
     end
   end
 
-  # Never trust parameters from the scary internet, only allow the white list through.
+  # Never trust parameters from the scary internet, only allow the white list
+  # through. Also sanitize the parameters to only contain UTF-8.
   def survey_result_params
-    params.require(:survey).
+    whitelisted_params = params.require(:survey).
       permit(SurveyResult::ALL_ATTRS.map(&:to_sym) + [:kind])
+    whitelisted_params.each do |k, v|
```

Comment:
Feel welcome to change the implementation. That said, I do think there is some value in replacing the character rather than stripping the character.